src/usr/lib/rsetup/tui/system/system.sh

```diff
@@ -151,14 +151,53 @@ You still need a valid bootloader on another location for the system to work.
 Are you sure you want to erase the bootloader?"
 }
 
-__system() {
+__system_target(){
+    radiolist_init
+
+    local target targets i
+    target="$(systemctl get-default)"
+    targets=(multi-user.target graphical.target)
```

Comment:
```suggestion
    current_target="$(systemctl get-default)"
    available_targets=(multi-user.target graphical.target)
```
还要判断target是否存在吧。比如CLI系统应该就没有graphical